makefile

```diff
@@ -158,6 +158,7 @@ $(gopath)/bin:
 $(buildDir)/.lintSetup:$(buildDir)/golangci-lint
 	$(gobin) get github.com/evergreen-ci/evg-lint/...
 	@mkdir -p $(buildDir)
+	@touch $@
```

Comment:
yay I think this fixes https://jira.mongodb.org/browse/EVG-7223

Reply:
Yeah I'll close that as a duplicate.